adoc-mode.el

```diff
@@ -46,6 +46,15 @@
 ;;; Variables:
 
 (require 'markup-faces) ; https://github.com/sensorflo/markup-faces
+
+(defface markup-native-code-face ;; TODO: This should go into `markup-faces.el'
```

Comment:
Rather we should delete markup-faces as a dependency and go back to adoc-mode specific faces.

Reply:
Yes, it is another issue.